wrappers/Gateway.ts

```diff
@@ -10,9 +10,60 @@ import {
     SendMode,
     toNano,
     TupleItemInt,
+    Slice,
 } from '@ton/core';
-import { evmAddressToSlice, loadHexStringFromBuffer, signCellECDSA } from '../tests/utils';
-import { Wallet as EVMWallet } from 'ethers'; // copied from `gateway.fc`
+import { Wallet } from 'ethers'; // copied from `gateway.fc`
+
+export function evmAddressToSlice(address: string): Slice {
```

Comment:
Consider moving these helpers to the bottom of the file

Reply:
07b0dec